src/helpers/parseNumbers.ts

```diff
@@ -10,17 +17,37 @@ BigNumber.NE = -20;
 // Converts a decimal number to the native representation (a large integer)
 export const decimalToNative = (value: BigNumber | number | string) => {
   const bigIntValue = new BigNumber(value);
-  const multiplier = new BigNumber(10).pow(TOKEN_DECIMALS);
+  const multiplier = new BigNumber(10).pow(AmplitudeDecimals);
+
+  return bigIntValue.times(multiplier);
+};
+
+export const decimalToStellarNative = (value: BigNumber | number | string) => {
+  const bigIntValue = new BigNumber(value);
+  const multiplier = new BigNumber(10).pow(StellarDecimals);
 
   return bigIntValue.times(multiplier);
 };
 
+export const fixedPointToDecimal = (value: BigNumber | number | string) => {
```

Comment:
why is this method called from fixedPoint?  It receives the same possible values than `decimalTo...` methods, and it treats  the value as a BigInt too. What it should receive then?

Reply:
There are now three different things we need to distinguish about numbers.
Our fee parameters (or most things that represent a percentage in spacewalk pallets) are represented by an UnsignedFixedPoint with `u128` as the inner type. This means that `1` or `100%` is represented by `1,000,000,000,000,000,000` ie 1e18, `10%` would be `100,000,000,000,000,000` etc. 
When it comes to balances, we need to consider the `decimals` parameter that the chain is using. On our chains that's `12` for each network. 
Lastly, Stellar assets only have a precision of 1e7. Thus we need a third conversion function for that.

It can be a bit confusing of course, but I think having three different helper functions with appropriate names helps.

Reply:
Okay, thanks a lot to clarify. I will then leave it like that then.